app/views/api/v1/app_identifier_visitor_configs/show.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.partial! 'api/v1/app_visitor_configs/show',
+  active_splits: @active_splits,
+  visitor_id: @visitor_id,
```

Comment:
seems like this line should be blowing up because we don't assign @visitor_id

Reply:
yeah, that's just classic "unset ivars return nil" behavior. womp. I fixed and added a test granular enough to catch it.

Reply:
yea, sorry, i just meant that i was surprised it didn't get caught in a test. coolcoolcool

Reply:
I don’t assert on the whole response because there’s a fair bit of factory defaults in there. It’s unusual to ship an ID solo like this, but I wanted to keep the assignment fetch explicit and in the controller (so nobody accidentally grabs visitor.assignments in the view layer and hurts themselves). That and the identifier controller was a copy paste of the visitor controller which was structurally immune to this issue since the @visitor_id was literally used to fetch assignments. so I didn’t intuitively test it. :(